### PR TITLE
Arun finished question 1

### PR DESCRIPTION
**Pull request by ArunRameshbabu**
Guys, please review the code

### CODE REVIEW

**Comment by Charlottepeddle12** [comment]
You did not do question 1

**Comment by ChristopherSquires** [comment]
Feel like Fresh Princes of Software Development would've been a pretty good team name

**Comment by ZCZDinn** [comment]
Code is good and Skywalkers is a good name

**Comment by ArunRameshbabu** [comment]
> You did not do question 1

Yup, YOU are supposed to do it!

File: Assignments/TheSkywalkers/question1.py
Comment (Charlottepeddle12) [inline Assignments/TheSkywalkers/question1.py:1]: Should type "Arun did NOT complete the question"

File: Assignments/TheSkywalkers/question1.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("Arun completed this assignment question")
```
Comment (ZCZDinn) [inline Assignments/TheSkywalkers/question1.py:1]: May the force be with you

File: Assignments/TheSkywalkers/question1.py
Comment (ChristopherSquires) [inline Assignments/TheSkywalkers/question1.py:1]: You didn't put it in a function like the qeustion asked

**Review by ChristopherSquires — APPROVED** [review]
Arun I left you a comment please respond

**Review by Charlottepeddle12 — COMMENTED** [review]
I do not approve of this team

**Review by ZCZDinn — APPROVED** [review]
Good Work!

**Review by Charlottepeddle12 — APPROVED** [review]

**Pull request merged**